sparkbench/pom.xml

```diff
@@ -121,6 +128,15 @@
         <spark.version>2.0.0</spark.version>
         <spark.bin.version>2.0</spark.bin.version>
       </properties>
+      <modules>
+        <module>micro</module>
+        <module>ml</module>
+        <module>websearch</module>
+        <module>graph</module>
+        <module>sql</module>
+        <module>streaming</module>
+        <module>structuredStreaming</module>
+      </modules>
```

Comment:
How can I get it compiled for spark 1.6 without the structured streaming module?